packages/desktop/src/renderer/components/widgets/channels/ChannelInput/ChannelInput.test.tsx

```diff
@@ -179,4 +196,137 @@ describe('ChannelInput', () => {
       </body>
     `)
   })
+
+  it('user can copy/paste images to message input', async () => {
+    window.ResizeObserver = jest.fn().mockImplementation(() => ({
+      observe: jest.fn(),
+      unobserve: jest.fn(),
+      disconnect: jest.fn()
+    }))
+    window.HTMLElement.prototype.scrollTo = jest.fn()
+    document.execCommand = jest.fn()
+    const mockHandleClipboardFiles = jest.fn()
+    const mockUnsupportedModalHandleOpen = jest.fn()
+
+    // preparing image data
+    const base64Image = 'iVBORw0KGgoAAAANSUhEUgAAAAcAAAAICAYAAAA1BOUGAAAAAXNSR0IB2cksfwAAABVJREFUCJljVFBU+c+AAzDhkhh6kgBGDQF0RFVIuQAAAABJRU5ErkJggg=='
+    const binaryString = window.atob(base64Image)
+    const len = binaryString.length
+    const bytes = new Uint8Array(len)
+    for (let i = 0; i < len; i++) {
+      bytes[i] = binaryString.charCodeAt(i)
+    }
+
+    const { store } = await prepareStore()
+    const factory = await getFactory(store)
+    await factory.create<ReturnType<typeof identity.actions.addNewIdentity>['payload']>(
+      'Identity',
+      { nickname: 'alice' }
+    )
+
+    const fileData = {
+      path: `data:image/jpeg;base64,${base64Image}`,
+      name: 'imageName',
+      ext: '.png',
+      arrayBuffer: bytes.buffer
+    }
+
+    const unsupportedFileModal: ReturnType<UseModalTypeWrapper<{
+      unsupportedFiles: FileContent[]
+      title: string
+      sendOtherContent: string
+      textContent: string
+      tryZipContent: string
+      handleOpen: jest.Mock<any, any>
+    }>['types']> = {
+      handleOpen: mockUnsupportedModalHandleOpen,
+      handleClose: jest.fn(),
+      open: false,
```

Comment:
Maybe it's stupid but I wish those props were ordered exactly like in generic (for keeping the code tidy and better readable)